tox.ini

```diff
@@ -102,7 +106,6 @@ depends =
     {coverage,functests}: tests
 commands =
     pip-sync-faster requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
-    dev: python bin/make_db
```

Comment:
`init_db` is now called by `supervisord-dev.conf` instead so that it runs in parallel with the other processes starting up